recipes/open62541/all/conanfile.py

```diff
@@ -0,0 +1,289 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans.model.version import Version
+from conans.errors import ConanInvalidConfiguration
+import glob
+import shutil
+import yaml
+
+
+class Open62541Conan(ConanFile):
+    name = "open62541"
+    license = "MPLv2"
+    exports_sources = [
+        "CMakeLists.txt",
+        "patches/**"
+    ]
+    exports = "submoduledata.yml"
+    homepage = "https://open62541.org/"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "open62541 is an open source and free implementation of OPC UA (OPC Unified Architecture) written in the common subset of the C99 and C++98 languages. The library is usable with all major compilers and provides the necessary tools to implement dedicated OPC UA clients and servers, or to integrate OPC UA-based communication into existing applications. open62541 library is platform independent. All platform-specific functionality is implemented via exchangeable plugins. Plugin implementations are provided for the major operating systems."
+    topics = (
+        "OPC UA", "open62541", "sdk", "server/client", "c", "iec-62541",
+        "industrial automation", "tsn", "time sensetive networks", "publish-subscirbe", "pubsub"
+    )
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "fPIC": [True, False],
+        "shared": [True, False],
+        "historize": ["Off", "On", "Experimental"],
```

Comment:
I assume you are a C/C++, feel free to mix types in python 😜 

```suggestion
        "historize": [True, False, "Experimental"],
```

Reply:
old habits die hard, I guess I can treat it as a std::variant<> :smiley: